### DISCUSSION TITLE
v5.0.0-beta10

### DISCUSSION THREAD

**Discussion by michaelliunsky** [michaelliunsky/niRvana-theme#23] [General]
- 支持 php8+
- 修复评论表情预览大小过大的 bug

<hr /><em>This discussion was created from the release <a href='https://github.com/michaelliunsky/niRvana-theme/releases/tag/v5.0.0-beta10'>v5.0.0-beta10</a>.</em>

**Comment by ffoocn** [comment]
> - 支持 php8+
> 
> - 修复评论表情预览大小过大的 bug
> 
> <hr /><em>This discussion was created from the release <a href='https://github.com/michaelliunsky/niRvana-theme/releases/tag/v5.0.0-beta10'>v5.0.0-beta10</a>.</em>

可以支持一键复制代码功能吗

**Reply by michaelliunsky** [reply]
已收到您的请求，将在以后版本中添加功能！

**Reply by michaelliunsky** [reply]
Hi，成功添加了功能，请测试！
